src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -299,6 +299,16 @@ class JiraUseCase {
         }
     }
 
+    private boolean mustRun(testIssue) {
+        return !(project.buildParams?.targetEnvironmentToken == 'P' &&
+            (testIssue.testType?.toLowerCase() in
+                [
+                    Project.TestType.ACCEPTANCE.toLowerCase(),
+                    Project.TestType.INTEGRATION.toLowerCase()
+                ]
```

Comment:
Yes, this is what the ticket said. Should I ignore everything that is not explicitly marked as INSTALLATION?

Reply:
Yes, the ticket only mentions those two (acceptance and integration). But I do wonder if we should consider unit here or not, just in case. @metmajer

Reply:
@albertpuente @jafarre-viewnext Unit tests are not executed in P, you can add it to the list.

Reply:
Ok, in this case it will be clearer to reverse the check and require only tests of type INSTALLATION